packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -179,6 +193,10 @@ ListItem.propTypes = {
    * Defines the `align-items` style property.
    */
   alignItems: PropTypes.oneOf(['flex-start', 'center']),
+  /**
+   * If `true`, the list item will be focused during the first mount.
```

Comment:
We should probably included that this is not just the first mount but also happens when the value flips from false to true.